libbeat/outputs/tidb_slow_query/schema.go

```diff
@@ -0,0 +1,191 @@
+		"Preproc_subqueries":            "double",
+		"Preproc_subqueries_time":       "double",
+		"Optimize_time":                 "double",
+		"Wait_TS":                       "double",
+		"Prewrite_time":                 "double",
+		"Wait_prewrite_binlog_time":     "double",
+		"Commit_time":                   "double",
+		"Get_commit_ts_time":            "double",
+		"Commit_backoff_time":           "double",
+		"Backoff_types":                 "varchar(64)",
+		"Resolve_lock_time":             "double",
+		"Local_latch_wait_time":         "double",
+		"Write_keys":                    "double",
+		"Write_size":                    "double",
+		"Prewrite_region":               "double",
+		"Txn_retry":                     "double",
+		"Cop_time":                      "double",
+		"Process_time":                  "double",
+		"Wait_time":                     "double",
+		"Backoff_time":                  "double",
+		"LockKeys_time":                 "double",
+		"Request_count":                 "double",
+		"Total_keys":                    "double",
+		"Process_keys":                  "double",
+		"DB":                            "varchar(64)",
+		"Index_names":                   "varchar(128)",
+		"Digest":                        "varchar(64)",
+		"Stats":                         "varchar(512)",
+		"Cop_proc_avg":                  "double",
+		"Cop_proc_p90":                  "double",
+		"Cop_proc_max":                  "double",
+		"Cop_proc_addr":                 "varchar(64)",
+		"Cop_wait_avg":                  "double",
+		"Cop_wait_p90":                  "double",
+		"Cop_wait_max":                  "double",
+		"Cop_wait_addr":                 "varchar(64)",
+		"Mem_max":                       "double",
+		"Disk_max":                      "double",
+		"Prev_stmt":                     "longtext",
+		"Query":                         "longtext",
+		"Is_internal":                   "tinyint(1)",
+		"Succ":                          "tinyint(1)",
+		"Plan_from_cache":               "tinyint(1)",
+		"Plan_from_binding":             "tinyint(1)",
+		"Prepared":                      "tinyint(1)",
+		"KV_total":                      "double",
+		"PD_total":                      "double",
+		"Backoff_total":                 "double",
+		"Write_sql_response_total":      "double",
+		"Plan":                          "longtext",
+		"Plan_digest":                   "varchar(128)",
+		"Backoff_Detail":                "varchar(4096)",
+		"Rocksdb_delete_skipped_count":  "double",
+		"Rocksdb_key_skipped_count":     "double",
+		"Rocksdb_block_cache_hit_count": "double",
+		"Rocksdb_block_read_count":      "double",
+		"Rocksdb_block_read_byte":       "double",
+	}
+	orderedColumn = make([]string, 0, len(slowQuerySQLType))
+	zone, _       = time.LoadLocation("Asia/Shanghai")
+)
+
+func init() {
+	for k := range slowQuerySQLType {
+		orderedColumn = append(orderedColumn, k)
+	}
+}
+
+const dateFormat = "2006-01-02"
```

Comment:
This is not used elsewhere, and is actually not reuseable for other functions. Better to put inline.